frontend/src/tests/components/Commons/CommonsList.test.js

```diff
@@ -80,14 +80,14 @@ describe("CommonsList tests", () => {
             i++;
         })
 
-        i = 0;
-        const ids = screen.getAllByTestId(/commonsCard-id/);
-        ids.forEach((id) => {
-            expect(id).toBeInTheDocument();
-            expect(typeof(id.textContent)).toBe('string');
-            expect(id.textContent).toEqual(commonsFixtures.threeCommons[i].id.toString());
-            i++;
-        })
+        // i = 0;
+        // const ids = screen.getAllByTestId(/commonsCard-id/);
+        // ids.forEach((id) => {
+        //     expect(id).toBeInTheDocument();
+        //     expect(typeof(id.textContent)).toBe('string');
+        //     expect(id.textContent).toEqual(commonsFixtures.threeCommons[i].id.toString());
+        //     i++;
+        // })
```

Comment:
This is just embarassing at this point.

Reply:
Sorry, professor, I did not know I was supposed to delete the comment. I just fixed that and the background array. Thank you for checking my PR